ruby/ql/test/library-tests/dataflow/local/local_dataflow.rb

```diff
@@ -74,5 +74,18 @@ def test_case x
   else
      x
   end
+
+  z = case source(1)
+  in 5 => b then sink(b) # $ hasTaintFlow=1
+  in a if a > 0 then sink(a) # $ hasTaintFlow=1
+  in [c, *d, e ] then [
+     sink(c), # $ hasTaintFlow=1
+     sink(d), # $ hasTaintFlow=1
+     sink(e)] # $ hasTaintFlow=1
+  in { a: f } then sink(f) # $ hasTaintFlow=1
+  in { foo: 1, g: } then sink(g) # $ hasTaintFlow=1
+  in { x: } then sink(x); x # $ hasTaintFlow=1
+end
+sink(z) # $ hasTaintFlow=1
```

Comment:
indent

Reply:
Why is there flow from `source(1)` to `z`?

Reply:
I added `; x` to `in { x: } then sink(x); x `: https://github.com/github/codeql/pull/7568/files#diff-387315d5b1966bf39afaae7daad4774dac1eb6700f3dcc475ee1bb6d3256c0e1R87

Reply:
Do you understand why there was flow prior to adding `; x`?

Reply:
There never was any. The expected output used to contain a line like which I hadn't spotted:
```
 failures
| local_dataflow.rb:89:11:89:28 | # $ hasTaintFlow=1 | Missing result:hasTaintFlow=1 |
```